Update PR template (#159)
WomenWhoCode/london: .github/PULL_REQUEST_TEMPLATE.md

<!--- Provide a general summary of your changes in the Title above -->

## Description

> <!--- Describe your changes in detail -->

Author: We can add it, but usually we know that because of the title: 
`feat: adding feature x `
`doc: improving README`
`fix: fixing the issue x`

> <!--- Describe your changes in detail -->

Author: ```suggestion
<!--- Describe your changes in detail -->

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [ ] Other

```

## Related Issue
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

> ## Motivation and Context

Author: Only if we say changes can be only done when we have an issue created. 
Otherwise, I think this makes sense.

> ## Motivation and Context

Reviewer: I see your point! yes, let's leave as it is then!

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## Screenshots
<!--  If you are chaging html, css or new resources it is mandatory to add screeshot. -->
<!--  Please add screenshot from *before* and *after* to simply the code review -->

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] I checked the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md) 
- [ ] I run locally and tested my changes

> - [ ] I add the website after I tested locally

Author: ```suggestion
- [ ] I have added a screenshot from the website after I tested it locally
```

> - [ ] I checked and followed the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md) 

Author: ```suggestion
- [ ] I checked and followed the [contributor guide](CONTRIBUTING.md) 
```
